<?xml version="1.0"?>
<launch>
  <arg name="width" default="400" />
  <arg name="height" default="400" />
  <arg name="save_images" default="false" />
  <arg name="use_gpu" default="true" />
  <arg name="loops" default="200"
      doc="the number of times to update before copying out and displaying an image, it is multiplied by 3 internally" />
  <arg name="cl_file" default="$(find cl_misc)/src/buffer.cl" />

  <node name="cl_buffer" pkg="cl_misc" type="cl_buffer"
      output="screen"
      args="$(arg cl_file)" >
    <param name="width" value="$(arg width)" />
    <param name="height" value="$(arg height)" />
    <param name="save_images" value="$(arg save_images)" />
    <param name="loops" value="$(arg loops)" />
    <param name="use_gpu" value="$(arg use_gpu)" />
  </node>
</launch>
